dspace-server-webapp/src/main/javadoc/overview.html

```diff
@@ -0,0 +1,32 @@
+<!--
+
+    The contents of this file are subject to the license and copyright
+    detailed in the LICENSE and NOTICE files at the root of the source
+    tree and available online at
+
+    http://www.dspace.org/license/
+
+-->
+<!DOCTYPE html>
+<!--
+Copyright 2024 Indiana University.
```

Comment:
Can we remove this? :)

Reply:
Well spotted!  Sorry, my IDE adds those automagically and I forgot to remove this one.  Done!